eng/imports/Packages.targets

```diff
@@ -1,129 +0,0 @@
-    <PackageReference Update="Microsoft.VisualStudio.ImageCatalog"                                    Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VisualStudio.Interop"                                         Version="17.4.0-preview-2-32826-307" />
-    <PackageReference Update="Microsoft.VisualStudio.ManagedInterfaces"                               Version="8.0.50728" />
-    <PackageReference Update="Microsoft.VisualStudio.RpcContracts"                                    Version="17.7.5-preview" />
-    <PackageReference Update="Microsoft.VisualStudio.SDK.EmbedInteropTypes"                           Version="15.0.36" />
-    <PackageReference Update="Microsoft.VisualStudio.Settings.15.0"                                   Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop"                     Version="3.2.2146" />
-    <PackageReference Update="Microsoft.VisualStudio.Shell.15.0"                                      Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VisualStudio.Shell.Design"                                    Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VisualStudio.Shell.Framework"                                 Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VisualStudio.Telemetry"                                       Version="17.7.47" />
-    <PackageReference Update="Microsoft.VisualStudio.TemplateWizardInterface"                         Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VisualStudio.Threading"                                       Version="17.7.15-preview" />
-    <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers"                             Version="17.7.15-preview" />
-    <PackageReference Update="Microsoft.VisualStudio.Utilities"                                       Version="17.7.36307-preview.2" />
-    <PackageReference Update="Microsoft.VisualStudio.Validation"                                      Version="17.6.11" />
-    <PackageReference Update="Microsoft.VisualStudio.XmlEditor"                                       Version="17.3.32804.24" />
-    <PackageReference Update="Microsoft.VSSDK.BuildTools"                                             Version="17.5.18-preview1" />
-    <PackageReference Update="System.Threading.Tasks.Dataflow"                                        Version="7.0.0" />
-    <PackageReference Update="Microsoft.VSDesigner"                                                   Version="17.0.0-preview-2-31223-026" />
-    <PackageReference Update="VsWebSite.Interop"                                                      Version="16.8.30523.219" />
-
-    <!-- https://github.com/dotnet/roslyn/issues/53877 -->
-    <PackageReference Include="StreamJsonRpc"                                                         Version="2.16.21-preview" />
```

Comment:
I"m confused about this part of the PR. This is an `Include` among a bunch of `Update`s. I'd expect us to need a new `Include` somewhere else in the repo for this to build.

It's also possible that the reason we needed to add an explicit package reference to `StreamJsonRpc` has been fixed, and that we can remove it altogether.